pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -50,6 +51,8 @@ var (
 	// while debouncing. Defaults to 10 seconds. If events keep
 	// showing up with no break for this time, we'll trigger a push.
 	DebounceMax time.Duration
+
+	_ = udpa.TypedStruct{}
```

Comment:
Can you add a comment about what this does? As documentation for the future and because I don't understand it right now either

Reply:
Done.